directives/if in location.txt

```diff
@@ -0,0 +1,21 @@
+status_zone
```

Comment:
This filename has spaces. Can we avoid that?

Reply:
replaced them with underscores